datajunction-server/datajunction_server/database/node.py

```diff
@@ -222,9 +223,24 @@ class Node(Base):  # pylint: disable=too-few-public-methods
 
     missing_table: Mapped[bool] = mapped_column(sa.Boolean, default=False)
 
+    history: Mapped[List[History]] = relationship(
+        primaryjoin="History.entity_name==Node.name",
+        order_by="History.created_at",
+        foreign_keys="History.entity_name",
+    )
+
     def __hash__(self) -> int:
         return hash(self.id)
 
+    @hybrid_property
```

Comment:
Wow. What is this?

Reply:
It's a sqlalchemy object property: https://docs.sqlalchemy.org/en/20/orm/extensions/hybrid.html#sqlalchemy.ext.hybrid.hybrid_property